[2023-08-31 17:43:02,988][ignite.engine.engine.Engine][INFO] - Engine run starting with max_epochs=1.
[2023-08-31 17:43:06,904][ignite.engine.engine.Engine][ERROR] - Current run is terminating due to exception: CUDA out of memory. Tried to allocate 4.69 GiB (GPU 0; 23.69 GiB total capacity; 15.91 GiB already allocated; 772.56 MiB free; 15.97 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
[2023-08-31 17:43:06,953][ignite.engine.engine.Engine][ERROR] - Engine run is terminating due to exception: CUDA out of memory. Tried to allocate 4.69 GiB (GPU 0; 23.69 GiB total capacity; 15.91 GiB already allocated; 772.56 MiB free; 15.97 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
